pyscriptjs/src/pyconfig.ts

```diff
@@ -44,7 +45,7 @@ export type PyScriptMetadata = {
 };
 
 const allKeys = Object.entries({
-    string: ['name', 'description', 'version', 'type', 'author_name', 'author_email', 'license'],
+    string: ['name', 'description', 'version', 'type', 'author_name', 'author_email', 'license', 'execution_thread'],
```

Comment:
`execution_realm` would be a bit more consistent with JavaScript lingo. But thread is probably fine since "realm" is a weird word.

Reply:
yes, `realm` is very weird.